pkgs/applications/version-management/git-and-tools/git-crypt/default.nix

```diff
@@ -12,16 +12,25 @@ stdenv.mkDerivation rec {
     inherit name;
   };
 
+  nativeBuildInputs = [ libxslt ];
```

Comment:
I think we're missing docbook deps
* https://github.com/AGWA/git-crypt/blob/master/man/git-crypt.xml#L1

Build fails like
```
xsltproc --param man.output.in.separate.dir 1 --stringparam man.output.base.dir man/ --param man.output.subdirs.enabled 1 --param man.authors.section.enabled 1 http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl man/git-crypt.xml
error : Resource temporarily unavailable
warning: failed to load external entity "http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl"
cannot parse http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl
```

For me with sandboxing

Reply:
Serves me right for not having that enabled yet.  Fixed.